airflow/providers/papermill/example_dags/example_papermill.py

```diff
@@ -82,10 +83,18 @@ def check_notebook(inlets, execution_date):
         parameters={"msgs": "Ran from Airflow at {{ execution_date }}!"}
     )
 
-    check_output = PythonOperator(
-        task_id='check_out',
-        python_callable=check_notebook,
-        inlets=AUTO
-    )
+    if version.startswith("1."):
+        # AUTO inlets do not work with Airflow 1.* series.
+        check_output = PythonOperator(
+            task_id='check_out',
+            python_callable=check_notebook,
+            input_nb="/tmp/out-{{ execution_date }}.ipynb"
```

Comment:
Python operator doesn't have an input_nb arg.

Reply:
Yes. Apparently it was a bug in the example_papermill before that I copied it from, Changed it to PapermilllOperator.